pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -665,6 +665,30 @@ def parse_args(args=None):
                               'https://developer.android.com/guide/'
                               'topics/manifest/'
                               'activity-element.html'))
+
+    ap.add_argument('--android-entrypoint', dest='android_entrypoint',
```

Comment:
This one seems fine, I don't think we could support the same thing any other way with such a simple api.

Minor: I'd probably pull out the 'org.kivy.android.PythonActivity' to a constant declaration somewhere.